services/teams/components/team-selection.tsx

```diff
@@ -2,14 +2,24 @@ import { fetchCurrentTeam, fetchUserTeams } from "../";
 import { TeamSelectionForm } from "./team-selection-form";
 
 export async function TeamSelection() {
-	const allTeams = await fetchUserTeams();
-	const currentTeam = await fetchCurrentTeam();
+	const _allTeams = await fetchUserTeams();
+	const _currentTeam = await fetchCurrentTeam();
+
+	const allTeams = _allTeams.map(({ id, name }) => ({
+		id,
+		name,
+	}));
+
+	const currentTeam = {
+		id: _currentTeam.id,
+		name: _currentTeam.name,
+	};
```

Comment:
:memo: It could have been extended to narrow it down with a select field in the fetch function, but this would have been implemented as a type definition using generics, which seemed a bit redundant. So it was decided to filter after the fetch.